lte/gateway/deploy/agw_install.sh

```diff
@@ -70,24 +77,64 @@ if [ "$KVERS" != "4.9.0-9-amd64" ]; then
   NEED_REBOOT=1
 fi
 
+# configure environment variable defaults needed for ansible
+ANSIBLE_VARS="PACKAGE_LOCATION=/tmp"
+if [ -n "${REPO_HOST}" ]; then
+    if [ -z "${REPO_PROTO}" ]; then
+        REPO_PROTO=http
+    fi
+    if [ -z "${REPO_DIST}" ]; then
+        REPO_DIST=stretch-stable
+    fi
+    if [ -z "${REPO_COMPONENT}" ]; then
+        REPO_COMPONENT=main
+    fi
+    # configure pkgrepo location
+    ANSIBLE_VARS="ovs_pkgrepo_proto=${REPO_PROTO} ovs_pkgrepo_host=${REPO_HOST} ovs_pkgrepo_path=${REPO_PATH} ${ANSIBLE_VARS}"
```

Comment:
I think this should even be inside the role directly in a var file and not be passed during execution of the playbook

Reply:
These values are set here to allow non-default values in the script -- was essential when debugging bare metal provisioning process.